rules/manager.go

```diff
@@ -146,6 +147,14 @@ func NewGroupMetrics(reg prometheus.Registerer) *Metrics {
 			},
 			[]string{"rule_group"},
 		),
+		groupSamples: prometheus.NewGaugeVec(
+			prometheus.GaugeOpts{
+				Namespace: namespace,
+				Name:      "rule_group_samples",
+				Help:      "The number of samples returned for given rule group.",
```

Comment:
This should mention that it was at the last evaluation, like the others do.

Reply:
Is `rule_group_last_evaluation_samples` ok?